handle/middlewares/headers_test.go

```diff
@@ -0,0 +1,28 @@
+package middlewares
+
+import (
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+const CookieHeader = "Set-Cookie"
+
+func TestAPIHeader(t *testing.T) {
+	now = func() time.Time {
+		return time.Date(2020, time.October, 9, 1, 2, 3, 4, time.UTC)
+	}
+
+	url := "http://localhost:3000/test"
+	expectedCookie := "gocity_api=http://localhost:3000/test; Path=/; Expires=Sat, 10 Oct 2020 01:02:03 GMT; Max-Age=90000"
```

Comment:
A better way to check cookies from recorder:

```
  request := &http.Request{Header: http.Header{"Cookie": recorder.HeaderMap["Set-Cookie"]}}
  cookie, err := request.Cookie(cookieKey)
  require.NoError(t, err)

  require.Equal(t, cookie.Value, "<URL HERE>")
```

Reply:
Have done the changes.
Could you please review